roles/distributive/tasks/main.yml

```diff
@@ -50,24 +10,5 @@
   with_items:
     - "{{ distributive_package }}"
     - "distributive-{{ current_role }}-{{ checklist_versions[current_role] }}"
-
-- name: create consul configuration directory
-  become: yes
-  file:
-    path: /etc/consul
-    state: directory
-    mode: 0770
-  tags:
-    - bootstrap
-    - consul
-    - distributive
-
-- name: configure consul health check
-  become: yes
-  template:
-    src: consul-healthcheck.json.j2
-    dest: /etc/consul/distributive-{{ current_role }}-check.json
-  tags:
-    - bootstrap
-    - consul
-    - distributive
+    - "https://bintray.com/artifact/download/siddharthist/mantl-testing/distributive-{{ current_role }}-consul-config-{{ consul_config_versions[current_role] }}.x86_64.rpm"
```

Comment:
This will have to be changed when https://github.com/asteris-llc/mantl-packaging/pull/81 is in.